api/Document.json

```
@@ -3779,10 +3779,12 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Nested calls are not supported from Firefox 82 (return <code>false</code>). See <a href='https://bugzil.la/1634262'>bug 1634262</a>."
```

style: if there's some condition for the note (e.g., it applies only to certain versions or operating system), I try to lead with it

```suggestion
              "notes": "From Firefox 82, nested calls are not supported (return <code>false</code>). See <a href='https://bugzil.la/1634262'>bug 1634262</a>."
```